src/app/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import styles from './Tabs.module.css';
+
+function getActiveClass(pathname: string) {
+  const location = useLocation();
+  if (!location.pathname.includes(pathname)) {
+    return;
+  }
+  return styles.tabs__active;
+}
+
+function Tabs(): JSX.Element {
+  return (
+    <div className={styles.tabs}>
```

Comment:
If this is a navigation, maybe `nav` makes more sense.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav